### DISCUSSION TITLE
use fleet service tokens

### DISCUSSION THREAD

**Discussion by adam-stokes** [elastic/e2e-testing#1066] [General]
It has been requested that we start using fleet service tokens, this discussion is for what this would look like and how we would utilize this in our e2e tests.

Usage on cli:

` --fleet-server-service-token=${token}`

And remove the user:pass from the --fleet-server-es

API:

`POST /api/fleet/service-tokens`

**Comment by blakerouse** [comment]
This should really be the only way it is used. We added `username:password` as a stop gap until service tokens where fully implemented. Now that we have them they should really be what is tested in e2e.

In 7.14 I would hope we can remove `username:password` and switch to only allow service tokens.

**Comment by adam-stokes** [comment]
https://github.com/elastic/e2e-testing/pull/1096
